modules/fedimint-ln-common/src/gateway_endpoint_constants.rs

```diff
@@ -17,6 +17,7 @@ pub const OPEN_CHANNEL_ENDPOINT: &str = "/open_channel";
 pub const CLOSE_CHANNELS_WITH_PEER_ENDPOINT: &str = "/close_channels_with_peer";
 pub const RECEIVE_ECASH_ENDPOINT: &str = "/receive_ecash";
 pub const ROUTING_INFO_V2_ENDPOINT: &str = "/routing_info";
+pub const CREATE_INVOICE_ENDPOINT: &str = "/create_invoice";
```

Comment:
Let's start migrating these to the lnv2 module

Reply:
Good point! Done.